NYPL/fedora_ingest_rails
# Tgr 48 indexer upgrades

## JIRA tickets:
- [Fedora Indexer Investigation(s)](https://newyorkpubliclibrary.atlassian.net/browse/TGR-48)

## Things Done:
- Upgraded Ubuntu to 22.04
- Upgraded Rails to 7.1.3
- Upgradesd Ruby to 3.3.22
- Upgraded various gems

## How to Test:
- Run all the rspecs
- Configure your local .env to have `REPO_SOLR_URL=http://10.225.132.218:8983/solr/repoapi`, get on the VPN, and accessinate your IP to https://nypl-dams-dev.nypl.org/index.php
- Post a raw body like
```
{
    "uuids": ["14b44ac0-c08a-0134-67c6-00505686a51c"],
    "test_mode": true
}
```
to `http://localhost:3000/ingest_requests` in postman or with CURL
- The request should process successfully (follow docker logs to ensure this or look at `http://localhost:3000/delayed_job`)
